### PR TITLE
Convert null payloadCreator to the identity

### LINKED ISSUE #169: createAction not accepting null as second argument
I have just tried to `v1.0.1` and I am getting this error stack.

Throws this error stack
```sh
Failed loading boot script: /home/stuart/freecodecamp/server/boot/react.js
Invariant Violation: Expected payloadCreator to be a function or undefined
    at invariant (/home/stuart/freecodecamp/node_modules/invariant/invariant.js:42:15)
    at createAction (/home/stuart/freecodecamp/node_modules/redux-actions/lib/createAction.js:30:27)
    at Object.<anonymous> (/home/stuart/freecodecamp/common/app/redux/actions.js:24:27)
    at Module._compile (module.js:570:32)
    at loader (/home/stuart/freecodecamp/node_modules/babel-register/lib/node.js:144:5)
    at Object.require.extensions.(anonymous function) [as .js] (/home/stuart/freecodecamp/node_modules/babel-register/lib/node.js:154:7)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Function._load (/home/stuart/freecodecamp/node_modules/pmx/lib/transaction.js:62:21)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/stuart/freecodecamp/common/app/routes/challenges/components/map/Challenge.jsx:12:1)
    at Module._compile (module.js:570:32)
    at loader (/home/stuart/freecodecamp/node_modules/babel-register/lib/node.js:144:5)
    at Object.require.extensions.(anonymous function) [as .jsx] (/home/stuart/freecodecamp/node_modules/babel-register/lib/node.js:154:7)
Invariant Violation: Expected payloadCreator to be a function or undefined
    at invariant (/home/stuart/freecodecamp/node_modules/invariant/invariant.js:42:15)
    at createAction (/home/stuart/freecodecamp/node_modules/redux-actions/lib/createAction.js:30:27)
    at Object.<anonymous> (/home/stuart/freecodecamp/common/app/redux/actions.js:24:27)
    at Module._compile (module.js:570:32)
    at loader (/home/stuart/freecodecamp/node_modules/babel-register/lib/node.js:144:5)
    at Object.require.extensions.(anonymous function) [as .js] (/home/stuart/freecodecamp/node_modules/babel-register/lib/node.js:154:7)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Function._load (/home/stuart/freecodecamp/node_modules/pmx/lib/transaction.js:62:21)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/stuart/freecodecamp/common/app/routes/challenges/components/map/Challenge.jsx:12:1)
    at Module._compile (module.js:570:32)
    at loader (/home/stuart/freecodecamp/node_modules/babel-register/lib/node.js:144:5)
    at Object.require.extensions.(anonymous function) [as .jsx] (/home/stuart/freecodecamp/node_modules/babel-register/lib/node.js:154:7)
```

The actions causing the error are in this format
```js
export const toggleMapDrawer = createAction(
  types.toggleMapDrawer,
  null,
  () => createEventMeta({
    category: 'Nav',
    action: 'toggled',
    label: 'Map drawer toggled'
  })
);
```

Any advice would be greatly appreciated

### PR DESCRIPTION
**Pull request by yangmillstheory**
Close #169.

### CODE REVIEW

File: src/createAction.js [hunk 1]
```diff
@@ -1,22 +1,27 @@
...
 import isUndefined from 'lodash/isUndefined';
+import isNull from 'lodash/isNull';
 import invariant from 'invariant';
 
 export default function createAction(type, payloadCreator = identity, metaCreator) {
   invariant(
-    isFunction(payloadCreator),
-    'Expected payloadCreator to be a function or undefined'
+    isFunction(payloadCreator) || isNull(payloadCreator),
+    'Expected payloadCreator to be a function, undefined, or null'
```
Comment (timche) [inline src/createAction.js]: Remove `,` => `undefined or null`.

File: src/createAction.js [hunk 2]
```diff
@@ -1,22 +1,27 @@
 import identity from 'lodash/identity';
 import isFunction from 'lodash/isFunction';
 import isUndefined from 'lodash/isUndefined';
+import isNull from 'lodash/isNull';
```
Comment (timche) [inline src/createAction.js]: I'd instead use [`isNil`](https://lodash.com/docs/#isNil), so we check for `null` **OR** `undefined`.

Comment (yangmillstheory) [inline src/createAction.js]: `undefined` is coerced at the parameter level, so I don't think we need to check for it downstream.

Comment (timche) [inline src/createAction.js]: {
        platform,
        production,
        destPath,
        verbose
    }

**Comment by yangmillstheory** [comment]
Do you want to manage the release (along with the #165 and possibly #166)?

**Pull request merged**